test/tstDetailsTreeConstruction.cpp

```diff
@@ -18,16 +18,13 @@
 
 #include <boost/test/unit_test.hpp>
 
-#include <algorithm>
 #include <array>
 #include <bitset>
 #include <functional>
 #include <limits>
 #include <sstream>
 #include <vector>
 
-#define BOOST_TEST_MODULE DetailsTreeConstruction
-
```

Comment:
Does this change belong here?

Reply:
This is arguable.  I am editing that file so I figure I might as well fix it.
I can do it elsewhere if you want me to.